Change (default commented) log filename as per Debian Policy section 10.8.

--- a/rsnapshot.conf.default.in
+++ b/rsnapshot.conf.default.in
@@ -113,7 +113,7 @@
 # If you enable this, data will be written to the file you specify. The
 # amount of data written is controlled by the "loglevel" parameter.
 #
-#logfile	/var/log/rsnapshot
+#logfile	/var/log/rsnapshot.log
 
 # If enabled, rsnapshot will write a lockfile to prevent two instances
 # from running simultaneously (and messing up the snapshot_root).
